src/components/Common/Product/ProductCreate/ProductCreateComponent.jsx

```diff
@@ -0,0 +1,150 @@
+    }
+  }
+
+  const handleInputChange = event => {
+    const inputText = event.target.value
+    if (inputText.length <= 30) {
+      setname(inputText)
+      setCharacterCount(inputText.length)
+    }
+  }
+
+  const handlePriceChange = event => {
+    const inputPrice = event.target.value.replace(/\D/g, '')
+    if (inputPrice.length <= 20) {
+      setPrices(inputPrice.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ','))
+      setSendPrice(parseInt(inputPrice))
+    }
+  }
+
+  const handleProductUpload = async event => {
+    const files = event.target.files
+    const fileArray = Array.from(files)
+
+    setSelectedImages(fileArray)
+
+    const imagePreviewsArray = await Promise.all(
+      fileArray.map(file => {
+        return new Promise((resolve, reject) => {
+          const reader = new FileReader()
+          reader.onload = () => resolve(reader.result)
+          reader.onerror = error => reject(error)
+          reader.readAsDataURL(file)
+        })
+      }),
+    )
+
+    setImagePreviews(imagePreviewsArray)
+
+    const formData = new FormData()
+    for (let i = 0; i < files.length; i++) {
+      formData.append('image', files[i])
+    }
+    const res = await uploadImages(formData)
+    console.log(res)
+    setImageUrl(`https://api.mandarin.weniv.co.kr/${res.data[0].filename}`)
+    setSelectedImage(files)
+  }
+
+  return (
+    <>
+      <section className={s.section}>
+        <form className={s.contentBox}>
+          <h2 className={s.title}>상품 등록페이지</h2>
+          <p className={s.subtitle}>-여러분의 상품을 보여주세요!-</p>
+          <section className={s.imageContainer}>
+            <h2>Step1. 상품 이미지 등록</h2>
+            <label htmlFor='productImg' className={s.a11yHidden}>
+              상품 이미지 업로드
+            </label>
+            <input
+              id='productImg'
+              type='file'
+              accept='image/*'
+              onChange={handleProductUpload}
+              className={s.imageInput}
+            />
+            <div className={s.imagePreviewContainer}>
+              {imagePreviews.map((preview, index) => (
+                <img key={index} src={preview} alt={`Preview ${index + 1}`} className={s.imagePreview} />
+              ))}
+            </div>
+          </section>
+
+          <section className={s.nameContainer}>
+            <h2>Step2. 상품명 등록</h2>
+            <label htmlFor='productName' className={s.a11yHidden}>
+              상품명 입력
+            </label>
+            <input
+              id='productName'
+              type='text'
+              className={s.name}
+              placeholder='상품명을 적어주세요.'
+              value={name}
+              onChange={handleInputChange}
+              maxLength={30}
+            />
+            <div className={s.counter}>{characterCount}/30</div>
+          </section>
+
+          <section className={s.priceContainer}>
+            <h2>Step3. 상품 가격 등록</h2>
+            <label htmlFor='productPrice' className={s.a11yHidden}>
```

Comment:
a11y-hidden은 reset.css에 정의되어 있기때문에
```suggestion
            <label htmlFor='productPrice' className="a11y-hidden">
```
다음과 같이 사용하시면 됩니다

Reply:
아하! reset.scss는 이렇게 사용하군요! 수정했습니다!